radar/2022-12-05/tailwind-css.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Tailwind CSS"
+ring: adopt
+quadrant: languages-and-frameworks
+featured: true
+---
```

Comment:
I would give this preference over chakra, but it will be handy to let people know when to use what, when to buy things to speed up the team, etc

Reply:
i have a diagram when to use what

Reply:
I agree tailwind is the go-to for most projects; I will document on the wiki when to use what.

Reply:
Ah ok. I didn't realise Tailwind and Chakra were alternatives to be honest. I have added now that we generally prefer Tailwind.